### PR TITLE
Cart btn wiggle

### PR DESCRIPTION
**Pull request by sineade**
:art:

Adding code to make the cart button in registries and manage membership bounce when there are items sitting in the cart for more than 10 seconds.

registries-bounce
manage-membership-bounce

### CODE REVIEW

File: app/assets/stylesheets/layout_components/gift-cards.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    border: none;
+  }
+
+  .small-link{
+    color: $white;
+    font-size: 0.7rem;
+    line-height: 15px;
+    display: block;
+  }
+  @include breakpoint(medium down){
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/layout_components/gift-cards.scss]: Missing a blank line above this one.

File: app/assets/stylesheets/layout_components/gift-cards.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  color: $white;
+  font-size: 0.7rem;
+  line-height: 15px;
+  display: block;
+  margin-bottom: $base-margin;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+#toggle-amount{
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/layout_components/gift-cards.scss]: Most of the styles here match the ones in `.toggle-card` above. Should the common parts be moved into an includable mixin, or combine the common rules `.toggle-card, #toggle-amount{ ... }`, or is a bit of repetition fine?

**Comment by MathieuGilbert** [comment]
💩

File: app/assets/stylesheets/layout_components/gift-cards.scss (hunk 2 above)
Comment (sineade) [inline app/assets/stylesheets/layout_components/gift-cards.scss]: good catch

**Comment by MathieuGilbert** [comment]
Did you mean to drop the margin bottom on that last change?

image

**Comment by sineade** [comment]
@MathieuGilbert ya, I checked and it wasn't needed. Thanks for checking.

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

**Comment by sineade** [comment]
:hankey: talked to kayt, going to change some things in this.

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Pull request merged**